src/screens/DeleteObservation/Popup.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import {
+  View,
+  UIManager,
+  findNodeHandle,
+  TouchableOpacity,
+  Image
+} from "react-native";
+
+/*eslint-disable */
+const dotsIcon = require("assets/three-dots-menu/dots-vertical-black.png");
+/* eslint-enable */
+
+export default class PopupMenu extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      icon: null
+    };
+
+    this.onPress = this.onPress.bind(this);
+  }
+
+  onError() {
+    console.log("Popup Error");
+    this.setState({ icon: null });
+  }
+
+  onPress() {
+    const { icon } = this.state;
+    const { actions, onPress } = this.props;
+
+    if (icon) {
+      UIManager.showPopupMenu(
+        findNodeHandle(icon),
+        actions,
+        this.onError,
+        onPress
+      );
+    }
+  }
+
+  render() {
+    const { onPress } = this;
+    /* eslint-disable */
+    const onRef = icon => {
+      if (!this.state.icon) {
+        this.setState({ icon });
```

Comment:
I don't think so that we need ref in a state. Could read about it more? https://reactjs.org/docs/refs-and-the-dom.html And it may help with eslint ;)

Reply:
Ref here better not to be deleted, because for UIManager.showPopupMenu() first parameter should be element's tag, not a React component.
Reference can be found here: [UIManagerModule.java, line 683](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java)